web/src/app/users/UserContactMethodList.tsx

```diff
@@ -78,7 +78,7 @@ export default function UserContactMethodList(
   if (!data?.user) return <ObjectNotFound type='user' />
   if (error) return <GenericError error={error.message} />
 
-  const contactMethods = data.user.contactMethods
+  const contactMethods = data?.user?.contactMethods ?? []
```

Comment:
Is this necessary with suspense? It _should_ be all-or-nothing (under the covers suspense throws if the data isn't loaded)

Reply:
I was getting console errors. I'll remove and try again.